src/CMakeLists.txt

```diff
@@ -236,16 +236,25 @@ endif ()
 
 target_link_libraries(ksnip ${DEPENDENCY_LIBRARIES})
 
-if (UNIX AND NOT APPLE)
-	install(TARGETS ksnip RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(TARGETS ksnip RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+if(WIN32)
+install(
+    CODE
+    "
+        include(InstallRequiredSystemLibraries)
+        include(BundleUtilities)
+	fixup_bundle(\"\${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/ksnip.exe\"  \"\" \"\")
```

Comment:
Those look like normal cmake calls, o they have to be in a quoted string or can they be defined outside?

Reply:
I kind of copied that from stackoverflow :) I believe it's called somehow at a later point when fixing up the bundle, but I have no clue. And it also does not get all the files, as you found out.